task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -2723,7 +2797,7 @@
     }
 
     .activityCharsCount.tooManyChars {
-      display: none;
+      //display: none;
```

Comment:
Why this modification is needed ?

Reply:
to make sure that the characters counter is always appeared